### PR TITLE
Product landing page done!

### PR DESCRIPTION
**Pull request by andcoding**

### CODE REVIEW

File: responsive-web-design/Build-product-landing-page/creative.css [hunk 1]
```diff
@@ -0,0 +1,311 @@
+* {
+    margin: 0;
+    box-sizing: border-box;
+    text-decoration: none;
+}
+
+body {
+    /* background-color: #fda635; */
+    background-image: linear-gradient( 115deg, #ed4b4bba, #285da2a6, #834343), url(https://thewindow.barneys.com/wp-content/uploads/2017/11/4600_readyforretouching.jpg);
```
Comment (migueliriano) [inline responsive-web-design/Build-product-landing-page/creative.css]: Why are we using an external URL for this image??

If for some reason the image on the page is deleted your website going to be affected, please download the image and include it into this project locally.

File: responsive-web-design/Build-product-landing-page/creative.css [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+#page-wrapper{
+    position: relative;
+}
+
+#header{
+    display: flex;
+    align-items: center;
+    position: fixed;
+    width: 100%;
+    /* box-shadow: 5px 5px 5px #de2222; */
```
Comment (migueliriano) [inline responsive-web-design/Build-product-landing-page/creative.css]: If we don't need this line of code, please remove it.

File: responsive-web-design/Build-product-landing-page/creative.css [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+    background-color: #f11a1ae6;
+    /* background-image: linear-gradient(45deg, black, transparent); */
```
Comment (migueliriano) [inline responsive-web-design/Build-product-landing-page/creative.css]: The same comment above.

File: responsive-web-design/Build-product-landing-page/creative.css [hunk 4]
```diff
@@ -0,0 +1,311 @@
...
+    display: inline-block;
+    padding: 10px;
+    text-align: center;
+}
+
+#header-img{    
+display: flex;    
+width: 150px;    
+height: 65px;
+}
```
Comment (migueliriano) [inline responsive-web-design/Build-product-landing-page/creative.css:37]: Add an empty line

File: responsive-web-design/Build-product-landing-page/creative.css [hunk 5]
```diff
@@ -0,0 +1,311 @@
...
+.logo{
+    display: inline-block;
+    padding: 10px;
+    text-align: center;
+}
+
+#header-img{    
+display: flex;    
+width: 150px;    
+height: 65px;
```
Comment (migueliriano) [inline responsive-web-design/Build-product-landing-page/creative.css]: Fix indentation here.

File: responsive-web-design/Build-product-landing-page/creative.css [hunk 6]
```diff
@@ -0,0 +1,311 @@
...
+}
+#header > nav > ul > li {
+    display: inline-block;
+    text-align: right;
+    padding: 1px 10px 10px 10px;
+    cursor: pointer;
+    margin-right: 30px;
+    font-size: large;
+    font-weight: 500;
+    /* color: cyan; */
```
Comment (migueliriano) [inline responsive-web-design/Build-product-landing-page/creative.css]: Delete this comment line if it is not needed.

File: responsive-web-design/Build-product-landing-page/creative.css [hunk 7]
```diff
@@ -0,0 +1,311 @@
...
+    height: 295px;
+    margin: 0px;
+}
+
+
+#nav-bar{
+    display: flex;
+    justify-content: flex-end;
+     width: inherit;
+    }
```
Comment (migueliriano) [inline responsive-web-design/Build-product-landing-page/creative.css]: Fix the indentation of `}`

File: responsive-web-design/Build-product-landing-page/creative.css [hunk 8]
```diff
@@ -0,0 +1,311 @@
...
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    height: 295px;
+    margin: 0px;
+}
+
+
```
Comment (migueliriano) [inline responsive-web-design/Build-product-landing-page/creative.css]: Delete this blank line

File: responsive-web-design/Build-product-landing-page/creative.css [hunk 9]
```diff
@@ -0,0 +1,311 @@
...
+.container{
+    max-width: 1000px;
+    width: 100%;
+    margin: 0 auto;
+}
+
+#features{
+    margin-top: -40px;
+    /* display: flex; */
+    /* flex-direction: column; */
```
Comment (migueliriano) [inline responsive-web-design/Build-product-landing-page/creative.css]: Delete these comments

File: responsive-web-design/Build-product-landing-page/creative.css [hunk 10]
```diff
@@ -0,0 +1,311 @@
...
+    flex-direction: column;
+    height: 109px;
+    padding: 5px;
+    font-weight: 700;
+}
+
+.grid .custom-design{
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
```
Comment (migueliriano) [inline responsive-web-design/Build-product-landing-page/creative.css]: I see you are repeating the same property `justify-content` and` flex-direction` and different selectors, can you verify if you can include these properties in the parent element?

I think that if you do this the child elements going to inherit the parent property

File: responsive-web-design/Build-product-landing-page/creative.css [hunk 11]
```diff
@@ -0,0 +1,311 @@
...
+}
+
+.btn{
+    padding: 0 20px;
+    height: 40px;
+    font-size: 1em;
+    font-weight: 900;
+    text-transform: uppercase;
+    border-radius: 3px;
+    /* background: transparent; */
```
Comment (migueliriano) [inline responsive-web-design/Build-product-landing-page/creative.css]: Delete this.

File: responsive-web-design/Build-product-landing-page/creative.css [hunk 12]
```diff
@@ -0,0 +1,311 @@
...
+#header nav ul li {
+    display: flex;
+    /* text-align: right; */
+    /* align-items: center; */
+    padding: 3px;
+    /* cursor: pointer; */
+    /* margin-right: 30px; */
+    /* font-size: large; */
+    font-weight: 100;
+    /* color: cyan; */
```
Comment (migueliriano) [inline responsive-web-design/Build-product-landing-page/creative.css]: Delete comments lines and fix indentation.

**Review by migueliriano — CHANGES_REQUESTED** [review]
A few comments

File: responsive-web-design/Build-product-landing-page/creative.css (hunk 10 above)
Comment (andcoding) [reply responsive-web-design/Build-product-landing-page/creative.css]: What happen here is that the parent element contains two child elements with different properties in regards to their position, size, margin among others. 

What I'll do is to rename the classes for the three elements to the right of the icon elements with the name(right-element), Which means that I'll have to remove two css that were unnecessary. 

Thanks for this observation.

File: responsive-web-design/Build-product-landing-page/creative.css [hunk 13]
```diff
@@ -0,0 +1,260 @@
+* {
+    margin: 0;
+    box-sizing: border-box;
+    text-decoration: none;
+}
+
+body {
+    background-image: linear-gradient( 115deg, #ed4b4bba, #285da2a6, #834343), url(backgroundimg.jpg);
```
Comment (migueliriano) [inline responsive-web-design/Build-product-landing-page/creative.css]: Please use a descriptive name for the background image, for example: `homepage_background.jpg`

File: responsive-web-design/Build-product-landing-page/creative.css [hunk 14]
```diff
@@ -0,0 +1,260 @@
...
+    text-align: center;
+    height: 295px;
+    margin: 0px;
+}
+
+#nav-bar{
+    display: flex;
+    justify-content: flex-end;
+    width: inherit;
+    }
```
Comment (migueliriano) [inline responsive-web-design/Build-product-landing-page/creative.css]: Fix the indentation of `}`

File: responsive-web-design/Build-product-landing-page/creative.css [hunk 15]
```diff
@@ -0,0 +1,260 @@
...
+
+#features .icon {
+   display: none;
+}
+
+#features .grid {
+  margin-bottom: 70px;
+}
+
+}
```
Comment (migueliriano) [inline responsive-web-design/Build-product-landing-page/creative.css]: Fix indentation

File: responsive-web-design/Build-product-landing-page/index.html [hunk 16]
```diff
@@ -0,0 +1,150 @@
...
+                <br>
+                <input id="submit" name="submit" type="submit" class="submit" value="Get Started">
+            </form>
+        </section>
+
+        <div class="container">
+
+            <section id="features">
+
+
```
Comment (migueliriano) [inline responsive-web-design/Build-product-landing-page/index.html]: Delete blank lines

File: responsive-web-design/Build-product-landing-page/index.html [hunk 17]
```diff
@@ -0,0 +1,150 @@
...
+        <header id="header">
+
+            <div class="logo">
+                <img id="header-img"
+                    src="timlogo.png"
+                    alt="timberland boots" width="200px" height="100px">
+            </div>
+            <!-- Write your comments here -->
+            <nav id="nav-bar">
+                <ul class="link-buttons" style="list-style: none;">
```
Comment (migueliriano) [inline responsive-web-design/Build-product-landing-page/index.html]: Please remove the inline style and apply the style using css

File: responsive-web-design/Build-product-landing-page/index.html [hunk 18]
```diff
@@ -0,0 +1,150 @@
...
+                        <i class="material-icons">local_shipping</i></div>
+                    <div class="men-shoes">
+                        <h2 id="men">Dress Boots & Shoes</h2>
+                        <p>Our dress shoes for men, including men’s oxfords, brogues and other men’s casual dress shoes,
+                            are designed for comfort in the office and out to happy hour.</p>
+                    </div>
+
+                </div>
+
+
```
Comment (migueliriano) [inline responsive-web-design/Build-product-landing-page/index.html]: Delete blank lines

File: responsive-web-design/Build-product-landing-page/index.html [hunk 19]
```diff
@@ -0,0 +1,150 @@
...
+                        <li>Lorem ipsum.</li>
+
+                    </ol>
+                    <button class="btn">Select</button>
+                </div>
+
+            </section>
+
+            <footer>
+                <ul style="list-style: none;">
```
Comment (migueliriano) [inline responsive-web-design/Build-product-landing-page/index.html]: Add this style using CSS

**Review by migueliriano — CHANGES_REQUESTED** [review]
@andcoding - Thank you for addressing the last comments, I left a few comments more.

File: responsive-web-design/Build-product-landing-page/creative.css [hunk 20]
```diff
@@ -0,0 +1,265 @@
...
+.product > .level{
+    background-color: rgb(8, 8, 8);
+    color: white;
+    padding: 15px 0;
+    width: 100%;
+    text-transform: uppercase;
+    font-weight: 800;
+}
+
+.product > h2{
```
Comment (migueliriano) [inline responsive-web-design/Build-product-landing-page/creative.css]: Replace this selector in all the cases that we don't need it.

File: responsive-web-design/Build-product-landing-page/creative.css [hunk 21]
```diff
@@ -0,0 +1,265 @@
...
+    box-sizing: border-box;
+}
+
+body {
+    background-image: linear-gradient( 115deg, #ed4b4bba, #285da2a6, #834343), url(homepage_background.jpg);
+    font-family: 'Open Sans', sans-serif;
+    color: black;
+    width: -webkit-fill-available;
+    height: fit-content;
+}
```
Comment (migueliriano) [inline responsive-web-design/Build-product-landing-page/creative.css:12]: Add an empty line

File: responsive-web-design/Build-product-landing-page/creative.css [hunk 22]
```diff
@@ -0,0 +1,265 @@
...
+
+.btn{
+    padding: 0 20px;
+    height: 40px;
+    font-size: 1em;
+    font-weight: 900;
+    text-transform: uppercase;
+    border-radius: 3px;
+    cursor: pointer;
+}
```
Comment (migueliriano) [inline responsive-web-design/Build-product-landing-page/creative.css:173]: Add blank line

File: responsive-web-design/Build-product-landing-page/creative.css [hunk 23]
```diff
@@ -0,0 +1,265 @@
...
+    padding: 0px 10px;
+}
+
+
+@media (max-width: 767px) {
+    .link-buttons{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
```
Comment (migueliriano) [inline responsive-web-design/Build-product-landing-page/creative.css]: Add indentation to the properties of this selector.

File: responsive-web-design/Build-product-landing-page/creative.css [hunk 24]
```diff
@@ -0,0 +1,265 @@
...
+   width: 90%;
+}
+
+#features .icon {
+   display: none;
+}
+
+#features .grid {
+   margin-bottom: 70px;
+}
```
Comment (migueliriano) [inline responsive-web-design/Build-product-landing-page/creative.css]: Add indentation to these selectors

File: responsive-web-design/Build-product-landing-page/index.html [hunk 25]
```diff
@@ -0,0 +1,139 @@
...
+
+<body>
+
+    <div id="page-wrapper">
+        <header id="header">
+
+            <div class="logo">
+                <img id="header-img"
+                    src="timlogo.png"
+                    alt="timberland boots" width="200px" height="100px">
```
Comment (migueliriano) [inline responsive-web-design/Build-product-landing-page/index.html]: Apply the width and height using css, please

**Review by migueliriano — CHANGES_REQUESTED** [review]
More comments

File: responsive-web-design/Build-product-landing-page/creative.css [hunk 26]
```diff
@@ -0,0 +1,276 @@
...
+
+#features .icon {
+   display: none;
+}
+
+#features .grid {
+    margin-bottom: 70px;
+    }
+}
```
Comment (migueliriano) [inline responsive-web-design/Build-product-landing-page/creative.css]: We have the indentation wrong yet, please fix these lines: https://github.com/andcoding/freecodecamp-courses/blob/8ff1e4a39837fdee9584390af70320357cf18876/responsive-web-design/Build-product-landing-page/creative.css#L203-L274

Using this code as an example:

```css
@media (max-width: 767px) {
    .link-buttons{
        display: flex;
        flex-direction: column;
        align-items: center;
        width: 100%;
     }
    .container{
        display: flex;
        flex-direction: column;
        text-align: center;
    }

    #nav-bar{
        display: flex;
        justify-content: flex-end;
        width: inherit;
    }
        
    #nav-bar li{
        text-align:  center;
        width: 100%;
        margin: auto;
    }

    #header nav ul li {
        display: flex;
        padding: 3px;       
        font-weight: 100;
    }

    #header {
        display: flex;
        flex-flow: wrap;
        width: 100%;
    }

    #form-elements{
        display: flex;
        justify-content: flex-end;
        text-align: -webkit-center;
        margin-bottom: 40px;
        height: 338px;
    }

    .logo {
        display: flex;
        justify-content: center;
        width: 100%;
    }

    #features {
        margin-top: 0px;
    }

    #pricing{    
       display: flex;        
       flex-direction: column;      
       align-items: center;
    }
    .product{
       width: 90%;
    }

    #features .icon {
       display: none;
    }

    #features .grid {
        margin-bottom: 70px;
    }
}
```

**Review by migueliriano — APPROVED** [review]
LGTM!!, thank you for these changes

**Comment by andcoding** [comment]
https://medium.com/free-code-camp/the-main-pillars-of-learning-programming-and-why-beginners-should-master-them-e04245c17c56


This is a short article I found insightful about the fundamental of
programming. Take at look at it.

**Pull request merged**